src/components/organisms/SectionQuotes.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { ReactElement, useEffect, useRef, useState } from 'react'
+import styles from './SectionQuotes.module.css'
+import { graphql, useStaticQuery } from 'gatsby'
+import { nanoid } from 'nanoid'
+import classNames from 'classnames/bind'
+import Container from '../atoms/Container'
+
+const cx = classNames.bind(styles)
+
+const query = graphql`
+  query QuotesQuery {
+    file(relativePath: { eq: "quotes.json" }) {
+      childContentJson {
+        title
+        quotes {
+          name
+          profilePicture
+          quote
+        }
+      }
+    }
+    profilePictures: allFile(
+      filter: {
+        extension: { regex: "/(jpg)|(jpeg)|(png)/" }
+        relativeDirectory: { eq: "quotes" }
+      }
+    ) {
+      edges {
+        node {
+          name
+          childImageSharp {
+            original {
+              src
+            }
+          }
+        }
+      }
+    }
+  }
+`
+interface QuotesContentData {
+  file: {
+    childContentJson: {
+      title: string
+      quotes: {
+        name: string
+        profilePicture: string
+        quote: string
+      }[]
+    }
+  }
+  profilePictures: {
+    edges: {
+      node: {
+        name: string
+        childImageSharp: { original: { src: string } }
+      }
+    }[]
+  }
+}
+
+interface Quote {
+  id: string
+  name: string
+  profilePicture: string
+  quote: string
+}
+
+export default function SectionQuotes(): ReactElement {
+  const data: QuotesContentData = useStaticQuery(query)
+  const { title } = data.file.childContentJson
+  const [fullQuotes, setFullQuotes] = useState<Quote[]>([])
```

Comment:
nitpick/opinion(non-blocking): I think just `[quotes, setQuotes]` would be enough.